ui/src/app/projects/projects.component.ts

```diff
@@ -79,13 +78,14 @@ export class ProjectsComponent implements OnInit {
     this.projectsObservable = Observable.fromPromise(
       this.projectsService.listProjects(filter)
         .then(listView => {
-          this.projects = listView.results ? listView.results : [];
+          this.projects = listView.results.sort() ? listView.results : [];
```

Comment:
nit: I realize it's equivalent but I'd probably split out the `sort()` onto a different line, or else put it on the latter part of this ternary expression. Putting it in the predicate makes the intent a little bit more ambiguous IMO.

Reply:
Moved outside ternary expression and added comment about sorting.